Contents/Sketch/typography.js

```diff
@@ -7,7 +7,10 @@ var applyTypography = function (newStyles, sharedStyles) {
     'justified': 3
   };
 
-  removeAllStyles();
```

Comment:
If you could get that working it'd be amazing. I've spent many many hours trying!